Prepare for auto-powered releases

asmacdo opened the pull request:
I followed the instructions for a project that is already released: https://intuit.github.io/auto/docs/welcome/getting-started#make-latest-release

Now, it appears to be configured correctly. 

```
./auto info                

Environment Information:

"auto" version: v11.1.5
"git"  version: v2.45.0
"node" version: v16.16.0

Project Information:

✔ Repository:      con/duct
✔ Author Name:     Austin Macdonald
✔ Author Email:    austin@dartmouth.edu
✔ Current Version: v0.0.1
✔ Latest Release:  v0.0.1

✔ Labels configured on GitHub project 

GitHub Token Information:

✔ Token:            [Token starting with ghp_]
✔ Repo Permission:  admin
✔ User:             asmacdo
✔ API:              
✔ Enabled Scopes:   repo, write:packages
✔ Rate Limit:       2862/5000
```

asmacdo commented:
I've also run `auto shipit --dry-run` and that looks good. I suppose we merge this, then I'll mark https://github.com/con/duct/pull/110/ as a "minor" update. (Its actually a breaking change, but I don't want auto to go to 1.0.0). When that merges, hopefully we see the magic!

jwodder commented:
@asmacdo 

* I see you're using the default `auto` label names of "major", "minor", and "patch".  If you ever enable Dependabot in this repository, this will lead to problems, so you may want to rename the labels to something like "semver-major", "semver-minor", and "semver-patch".  See https://github.com/datalad/datalad-installer/issues/175 for more information.
* `release.yml` is currently configured to only run on workflow dispatch, i.e., manually, rather than when a PR is merged.  Is that intentional?
* `.autorc` currently sets `onlyPublishWithReleaseLabel` to `true`, so releases require a PR to be marked with "release", and the "skip-release" label is unnecessary.  Is that intentional?

asmacdo commented:
ty @jwodder

> release.yml is currently configured to only run on workflow dispatch, i.e., manually, rather than when a PR is merged. Is that intentional?

No that was not intentional, good catch.

> .autorc currently sets onlyPublishWithReleaseLabel to true, so releases require a PR to be marked with "release", and the "skip-release" label is unnecessary. Is that intentional?

I prefer explicitly ask for a release, I've deleted the skip-release label

I've updated the auto labels to use the semver prefix and https://github.com/con/duct/labels to match

yarikoptic reviewed (approved):
if you think it is ready -- fire away and see if works! ;)
